opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -1364,25 +1685,24 @@ const std::string& FieldProps::canonical_fipreg_name(const std::string& fipreg)
         : fipreg;
 }
 
-/*
-  This function generates a new ACTNUM vector.The ACTNUM vector which is
-  returned is joined result of three different data sources:
-
-     1. The ACTNUM of if the grid which is part of this FieldProps structure.
-
-     2. If there have been ACTNUM operations in the DECK of the type:
-
-        EQUALS
-            ACTNUM 0 1 10 1 10 1 3 /
-        /
-
-     3. Cells with PORV == 0 will get ACTNUM = 0.
-
-  Observe that due to steps 2 and 3 the ACTNUM vector returned from this
-  function will in general differ from the internal ACTNUM used in the
-  FieldProps instance.
-*/
-std::vector<int> FieldProps::actnum() {
+// This function generates a new ACTNUM vector.The ACTNUM vector which is
+// returned is joined result of three different data sources:
```

Comment:
Suggested reformulation: The returned ACTNUM vector is a joined result from three different data sources: